config/sample_application.yml

```diff
@@ -19,3 +19,5 @@ PUSHER_CLUSTER:
 TWITCH_CLIENT_ID:
 TWITCH_CLIENT_SECRET:
 TWITCH_WEBHOOK_SECRET:
+
+REDISCLOUD_URL:
```

Comment:
now that `REDISCLOUD_URL` is not part of the Envfile, we might want to remove it from here

Reply:
I actually added it to here since we no longer have it in the ENV file so people know to add it if they are using Heroku to hook up Redis. But I guess we can live without it here.